src/identity/components/GlobalHeader/GlobalHeaderDropdown/GlobalHeaderTypeAheadMenu.tsx

```diff
@@ -0,0 +1,162 @@
+import React, {ChangeEvent} from 'react'
+import {Dropdown, Input} from '@influxdata/clockface'
+import {FixedSizeList as List} from 'react-window'
+import classnames from 'classnames'
+import {TypeAheadMenuItem} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown'
+
+type Props = {
+  defaultText?: string
+  defaultSelectedItem?: TypeAheadMenuItem
+  style?: React.CSSProperties
+  typeAheadPlaceHolder: string
+  typeAheadMenuOptions: TypeAheadMenuItem[]
+  onSelectOption: (item: TypeAheadMenuItem) => void
+}
+
+type State = {
+  searchTerm: string
+  queryResults: TypeAheadMenuItem[]
+  selectedItem: TypeAheadMenuItem
+}
+
+const isBlankStr = (str: string): boolean => str.trim().length === 0
```

Comment:
Yes, so if the org name / account name is `''` then we can use a default text.

Reply:
After an offline chat with @wdoconnell ,  we think that org name should be shown as is, even if its blank because that is what the user chose it to be. It also sounds like a crucial element of the app and if we allow less possibilities for bugs (by removing unnecessary logic) that will be better.

Reply:
👍